gcp/opentelemetry-demo-workload-identity.yaml.gotmpl

```diff
@@ -23,3 +23,57 @@ opentelemetry-collector:
     annotations:
       "iam.gke.io/gcp-service-account": "opentelemetry-demo@{{ . }}.iam.gserviceaccount.com"
 {{ end }}
+
```

Comment:
This file is named `opentelemetry-demo-workload-identity.yaml.gotmpl` - I had intended each feature to be in its own template. Can you put these new sections into separate file(s)?